pkg/mtbroker/common.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
I'm not sure, but I am no longer a fan of "common" files specially in go, file name IMHO should be more descriptive. Maybe here it can be `health` or `healthcheck`

Reply:
`healthcheck` or `health` would be great without the `Sender` interface, I agree `common.go` isn't a comprehensive name, though. If we can find a better place for the `Sender` interface we can rename it to `health.go`.

Reply:
As i said in the other comment, i think this file should not live at all here, but in an healthcheck package i think